client/src/components/ReviewContent/Review.jsx

```diff
@@ -41,28 +43,38 @@ const Review = ({ review }) => {
     }],
   };
 
+  function renderStars(num) {
+    const stars = Array.apply(null, new Array(5))
+    return stars.map((star, index) => (
+      <img
+        alt="star"
+        style={{ width: '15px' }}
+        src={index < num ? STAR_IMAGE : EMPTY_STAR_IMAGE}
+      />
+    ));
+  }
+
+  function recommendationStringify (bool) {
+    // function that will turn boolean into string
+  }
+
   return (
     <div className={styles.block}>
       <div className={styles.inline_block_bigger}>
-        <span>
-          {`${review && review.starRating} stars`}
-          &nbsp;&nbsp;
+        <span className={styles.tabs}>
+          {renderStars(`${review && review.starRating}`)}
+          {/* {`${review && review.starRating} stars`} */}
         </span>
-        <span>
+        <span className={styles.tabs}>
           {review && review.reviewUsername}
-          &nbsp;&nbsp;
         </span>
         <span>
-          ·&nbsp;
-          {review && review.reviewDate}
-          &nbsp;
-          months ago
+          {`${review && review.reviewDate} months ago`}
```

Comment:
Can you explain it to me why you are using && here? I mean just in general in your code. It would be great if you can describe it to me. Because I don't understand 🥺